src/package.xml

```diff
@@ -300,6 +308,13 @@
         <members>Hierarchy_Settings__c.Simple_Address_Change_Treated_as_Update__c</members>
         <members>Hierarchy_Settings__c.Store_Errors_On__c</members>
         <members>Hierarchy_Settings__c.Student_RecType__c</members>
+        <members>Plan_Requirement__c.Category__c</members>
```

Comment:
Do we need to list the fields individually if we're including the entire custom object? Maybe this is a translation requirement I'm not aware of...

Reply:
Yeah i was thinking the same thing. I remember the build failed if I wasn't adding the custom fields.